build/commands/lib/test.js

```diff
@@ -16,6 +16,7 @@ const getTestsToRun = (config, suite) => {
     } else {
       testsToRun.push('bin/run_brave_public_test_apk')
     }
+    testsToRun.push('crypto_unittests')
```

Comment:
we don't want to do this, tests should be added by adding them to brave/test/BUILD.gn

Reply:
It is a separate executable but now with https://github.com/brave/brave-core/pull/9913/commits/1349e79853b3daec099801b0870c88a524569b4a
we just need `"//crypto/symmetric_key_unittest.cc` and it will be built with `brave_unittests`